### DISCUSSION TITLE
New classes are not generated automatically

### DISCUSSION THREAD

**Discussion by VeselyT** [tailwindlabs/tailwindcss#9500] [Help]
Hello, 

we have been using tailwind with angular for some time now and it was working fine. 
Not sure at what point it broke but now new classes are generated only on the first start (I disabled webpack cache to see if was causing this issue or not).

We are using: 
Angular 14.2
Tailwind 3.1.8
 
Our `tailwind.config.js` is:
 ```
 ...

module.exports = {
  content: [
    './apps/frontend/src/**/*.{html,ts}',
    './libs/**/*.{html,ts}',
  ],
  ...
 };
 ```
 
`postcss.config.js`: 

```
  module.exports = {
    plugins: {
      tailwindcss: { config: './tailwind.config.js' },
      autoprefixer: { remove: false },
    },
  };

```

Any idea what could be misconfigured?

**Comment by adamwathan** [comment]
Can you create a reproduction we can clone and play with? Will greatly increase the chance of someone helping, having to create a project from scratch and set it all up means this will probably not get looked at for a while.

**Reply by VeselyT** [reply]
We were able to find the issue after all. In one nested folder we have package.json that is symbolic link 
![image](https://user-images.githubusercontent.com/6859743/194565967-3719c858-5e73-475f-91b4-601d96ccffb8.png)
and this seems to break the generation of new classes. I'll try to create some minimal reproduction repository next week and I'll eventually create an issue for that.
